lib/connection.js

```diff
@@ -193,6 +197,10 @@ class Connection extends EventEmitter {
   _heartbeatCallback(interval) {
     this.transport.send('{}');
     this.setTimeout(interval, this._heartbeatCallbackInstance);
+
+    if (process.env.NODE_ENV !== 'production') {
+      this.emit('sentPacket', this._heartbeatMessage);
```

Comment:
Yes, but it'll be strange if people will have test failing if they start them in console with NODE_ENV set to 'production', from maybe, the start of their server or anything.